src/ResourceManager/Profile/Commands.Profile/Models/ModelExtensions.cs

```diff
@@ -31,5 +35,27 @@ internal static AzureSubscription ToAzureSubscription(this Subscription other, A
                 context.Tenant.Id.ToString());
             return subscription;
         }
+
+        public static List<AzureTenant> MergeTenants(
```

Comment:
This method name is a bit misleading - we are not merging tenants here - we are setting/overriding them